onlineteamscan_v2_fe/pages/teams/index.vue

```diff
@@ -223,7 +223,7 @@ export default {
     },
     async confirmDeleteTeam() {
       this.loading = true
-      await this.$axios.delete(`teams/${this.editedTeam.id}`)
+      await this.$axios.delete(`teams/${this.editedTeam.id}`) // REV: error handling could be added
```

Comment:
Wat gebeurd er nu als de Axios call faalt? 

Het is bv mogelijk om de error pages te laten zien. https://nuxtjs.org/blog/understanding-how-fetch-works-in-nuxt-2-12#5-error-handling